DEVELOPMENT.md

```diff
@@ -1,24 +1,117 @@
 # Development
 
-## Feature Release Flow
-
-### Branching Strategy
+Please carefully read our [Coding Conventions](./CodingConventions.md).
+
+## Branching Convention
+
+**This convention is _binding_ for all PR requests in this repo!**
+Deviating from this convention has a significant risk of compromising mainnet. 
```

Comment:
Is it worth mentioning that if a deviation already occurred, or if a deviation has to be made due to to the current scenario we are in (unknown future scenario), the deviation should be clearly communicated to a specific channel?

Reply:
thanks. Addressed your suggestion in commit https://github.com/onflow/flow-go/pull/6177/commits/c0d09091b00ed01d17de7a1063714e1cfc011f93. I didn't mention a specific channel, because that might change to quickly. Nevertheless, I think its important to mention the responsibility for clear communication.